internal/restic/snapshot_policy.go

```diff
@@ -90,7 +90,9 @@ func ApplyPolicy(list Snapshots, p ExpirePolicy) (keep, remove Snapshots) {
 		{p.Yearly, y, -1},
 	}
 
+	var nr int
 	for _, cur := range list {
```

Comment:
We can use the index from the list here:
```
for nr, cur := range list {
```

Then we don't need another variable.